src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -82,8 +84,10 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
     window: any = window;
 
     componentDidMount() {
+        const { versionUpsellExperience, onLoad } = this.props;
         this.initialize();
         this.fetchData();
+        onLoad('preview', 'versions', { versionUpsellExperience });
```

Comment:
Should we wait until the data has loaded? If so, let's update `fetchData` to return its promise and then do this:

```suggestion
        this.fetchData()
            .then(() => {
                onLoad('preview', 'versions', { versionUpsellExperience });
            });
```

Reply:
ah, good idea. Will update